### DISCUSSION TITLE
How enable plugin management with RabbitMQContainerTest

### DISCUSSION THREAD

**Discussion by exile69000** [testcontainers/testcontainers-dotnet#1047] [Q&A] [answered]
Hi,

I would like to enable plugin management in RabbitMQContainerTest but it's not working. 

I try this : 
`
RabbitMqcontainer = new RabbitMqBuilder()
 .WithImage(RabbitMqImage)
 .WithPortBinding(5672, true)
 .WithPortBinding(15672, true)
 .WithCommand("rabbitmq-plugins enable rabbitmq_management")
 .WithUsername(DefaultUsername)
 .WithPassword(DefaultPassword)
.Build()`

Java testcontainer have a method : withPluginsEnabled 

Thanks for your help

**Comment by HofmeisterAn** [comment, marked as answer]
I think the module's base image (`rabbitmq:3.11`) does not support plugins. You need to override the module's image too:

```csharp
.WithImage("rabbitmq:3-management")
.WithCommand("rabbitmq-server", "rabbitmq-plugins", "enable", "rabbitmq_management")
```

**Reply by exile69000** [reply]
Great it's working :

```c#
RabbitMqcontainer = new RabbitMqBuilder().WithImage("rabbitmq:3-management")
                                        .WithPortBinding(5672, true)
                                        .WithPortBinding(15672, true)
                                        .WithCommand("rabbitmq-server", "rabbitmq-plugins", "enable", "rabbitmq_management")
                                        .Build();
```
